update join model scores functions to allow both or one of the csvs to be passed

2320sharon opened the pull request:
I updated these functions so that the user can pass one or both of the csvs with the model scores. I also renamed the columns from model_scores to 'classifer_model_score' and 'segmentation_model_score'

2320sharon committed:
update join model scores functions to allow both or one of the csvs to be passed

dbuscombe-usgs commented:
Do we need 4 variables to make this really useful?
'Image_suitability_model_score' 
'Image_suitability_model_threshold'
'Segmentation_suitability_model_score'
'Segmentation_suitability_model_threshold'

2320sharon committed:
Merge branch 'main' into join_model_scores_update

2320sharon committed:
add threshold used with both models

2320sharon commented:
Alright I added the thresholds for the classification and segmentation models. I also updated coastseg to save the thresholds used to the classification csv file. You can find these changes on this branch in coastseg : https://github.com/SatelliteShorelines/CoastSeg/tree/add_model_scores
